src/IB/IBFEMethod.cpp

```diff
@@ -2682,23 +2693,9 @@ IBFEMethod::spreadTransmissionForceDensity(const int f_data_idx,
     const bool integrate_tangential_force = d_split_tangential_force;
     if (!integrate_normal_force && !integrate_tangential_force) return;
 
-    const int coarsest_ln = 0;
-    const int finest_ln = d_hierarchy->getFinestLevelNumber();
-    VariableDatabase<NDIM>* var_db = VariableDatabase<NDIM>::getDatabase();
-
-    // Make a copy of the Eulerian data.
-    Pointer<hier::Variable<NDIM> > f_var;
-    var_db->mapIndexToVariable(f_data_idx, f_var);
-    const int f_copy_data_idx = var_db->registerClonedPatchDataIndex(f_var, f_data_idx);
```

Comment:
Since we already use a temporary variable in `spreadForce` we can completely remove this extra patch index here.